Tests/PointFreeTests/AccountTests/AccountTests.swift

```diff
@@ -73,13 +74,13 @@ final class AccountTests: TestCase {
   }
 
   func testAccount() async throws {
-    Current = .teamYearly
+    await withDependencyValues {
+      $0.teamYearly()
+    } operation: {
```

Comment:
Pretty much all test changes should fall into this simple pattern: opening up `withDependencyValues`, moving `Current` mutation inside the first closure, changing to `$0`, and then moving assertions to the `operation` closure.